t/72-autodie.t

```diff
@@ -9,7 +9,7 @@ eval "use autodie; 1"
 
 print "autodie $autodie::VERSION $INC{'autodie.pm'}\n";
 
-plan skip_all => "Currently a developer-only test" unless -d '.svn';
+plan skip_all => "Currently a developer-only test" unless -d '../.git';
```

Comment:
this doesn't seem right. wouldn't .git be in the current working directory? in which case we'd remove `../`